espnet/ap.go

```diff
@@ -17,7 +30,10 @@ package espnet
 */
 import "C"
 
-import _ "compat/freertos"
+import (
+	"unsafe"
+	_ "compat/freertos"
```

Comment:
These two should be reversed.

```suggestion
	_ "compat/freertos"
	"unsafe"
```